R/calcFE.R

```diff
@@ -28,7 +27,7 @@ calcFE <- function(source = "IEA", scenario_proj = "SSP2") {
     colnames(map) <- gsub("io", "names_in", colnames(map))
 
     # Give description
-    descript <- "IEA Final Energy Data based on 2017 version of IEA Energy Balances"
+    descript <- "IEA Final Energy Data based on 2022 version of IEA Energy Balances"
```

Comment:
this is only true for *_latest IEA data, right?

Reply:
no, it is true for the current default, i.e the description is accurate for inputdata generation. latest would be 2023. i decided not to make it conditional (to avoid another if-clause) in all places where we add a description and accept that the description is inaccurate when using it solely for generating the historical.mif.